### ISSUE TITLE
Issue with using the dropdown with voiceOver on an IOS physical device.

### ISSUE THREAD

**Issue by vytch** [GeekyAnts/NativeBase#4745]
### Description

When we enable voiceOver and we open the dropdown, we cannot swipe beyond a subset of the content in the background.

### CodeSandbox/Snack link

http://snack.expo.dev/

### Steps to reproduce

1. Clone the nativebase repo
2. yarn
3. Cd example
4. yarn
5. yarn ios
6. Download the expo client en the iphone and go to exp://192.168.1.5:19000
7. Enable voiceOver
8. Focus on the dropdown and open it.
9. Swipe a few times.
10. It seems we are trapped inside a subset of the page behind the actionSheet.

### NativeBase Version

3.3.1

### Platform

- [ ] Android
- [ ] CRA
- [ ] Expo
- [X] iOS
- [ ] Next

### Additional Information

We have tried to programmatically focus on the first item in the actionSheet. This works fine with Android but not with IOS.

**Comment by vytch** [comment]
Hi

We have made an investigation.
It seems that the issue comes from the modal that actionSheet is using.
By using the modal from react native, it seems that it works a lot better on IOS with voiceover.

Here is the fork we have been working on:
https://github.com/GeekyAnts/NativeBase/compare/master...Unibrighter:tech-spike-nab-iOS-actionsheet

I hope it helps
